src/libraries/Microsoft.PowerFx.Core/Errors/BaseError.cs

```diff
@@ -111,12 +111,17 @@ internal abstract class BaseError : IDocumentError
 
         private static readonly string HowToFixSuffix = "_HowToFix";
 
+        internal BaseError(IDocumentError innerError, Exception internalException, DocumentErrorKind kind, DocumentErrorSeverity severity, ErrorResourceKey errKey, CultureInfo locale, params object[] args)
+            : this(innerError, internalException, kind, severity, errKey, textSpan: null, sinkTypeErrors: null, locale, args: args)
+        {
+        }
+
         internal BaseError(IDocumentError innerError, Exception internalException, DocumentErrorKind kind, DocumentErrorSeverity severity, ErrorResourceKey errKey, params object[] args)
-            : this(innerError, internalException, kind, severity, errKey, textSpan: null, sinkTypeErrors: null, args: args)
+            : this(innerError, internalException, kind, severity, errKey, textSpan: null, sinkTypeErrors: null, CultureInfo.CurrentCulture, args: args)
```

Comment:
>CultureInfo.CurrentCulture [](http://example.com/codeflow?start=112&length=26)

can we avoid this - and require the updates?

Reply:
Not all. This is also being called from ErrorContainer which doesn't contain culture information.
https://github.com/microsoft/Power-Fx/blob/60838bf8b7024a15402ae44fdbf0f3f30a80db5e/src/libraries/Microsoft.PowerFx.Core/App/ErrorContainers/ErrorContainer.cs#L114